### PR TITLE
7.0 Add module workcenter group

### PR DESCRIPTION
**Pull request by bealdav**
Ready for merge, thanks for the review

### CODE REVIEW

**Comment by rvalyi** [comment]
:+1:  LGTM kind of basic but still good for compatibility to build upon the same basis.

**Comment by bealdav** [comment]
@rvalyi ,  right some other small mrp modules like this will comes these days to serve more advanced features

**Comment by sebastienbeau** [comment]
:+1: Simple module but usefull one ;)

**Comment by sebastienbeau** [comment]
Can you use the OCA template? And add OCA as Maintainer of the module?

File: mrp_workcenter_group/__init__.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+###############################################################################
+
+from . import (
```
Comment (clonedagain) [inline mrp_workcenter_group/__init__.py]: why a tuple?

File: mrp_workcenter_group/__openerp__.py [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+    'description': """
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
+MRP Workcenter Group
+====================
+
+* Add a new model: Workcenter Groups
+* Add a many2one field 'Group' in workcenter view form based on this model
+* Define a new 'Group by' entry named 'Group' in search view
```
Comment (clonedagain) [inline mrp_workcenter_group/__openerp__.py]: Users would rather have this in a human language :) Tell us what it's for, not how it's done.
If we want the technical details we have the code and the ORM's introspection.

File: mrp_workcenter_group/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+Usage
+=====
+
+To use this module, you need to go to:
+
+ * Manufacturing > Configuration > Workcenter Groups
+
+For further information, please visit:
+
+ * https://www.odoo.com/forum/help-1
```
Comment (clonedagain) [inline mrp_workcenter_group/__openerp__.py]: The URL is wrong isn't it?

File: mrp_workcenter_group/__openerp__.py [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+   :target: http://odoo-community.org
+
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization
+whose mission is to support the collaborative development of Odoo features
+and promote its widespread use.
+
+To contribute to this module, please visit http://odoo-community.org.
+
```
Comment (clonedagain) [inline mrp_workcenter_group/__openerp__.py]: empty line

File: mrp_workcenter_group/__openerp__.py [hunk 5]
```diff
@@ -0,0 +1,104 @@
...
+""",
+    'website': 'http://www.akretion.com/',
+    'data': [
+        'workcenter_view.xml',
+    ],
+    'demo': [
+    ],
```
Comment (clonedagain) [inline mrp_workcenter_group/__openerp__.py:90]: You could remove the empty lists for readability.

File: mrp_workcenter_group/__openerp__.py [hunk 6]
```diff
@@ -0,0 +1,104 @@
...
+    'external_dependencies': {
+        'python': [],
+    },
+    'license': 'AGPL-3',
+    'tests': [],
+    'installable': True,
+    'auto_install': False,
```
Comment (clonedagain) [inline mrp_workcenter_group/__openerp__.py:94]: Not useful either

File: mrp_workcenter_group/security/model_security.xml [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+    <data>
+
+<!-- Multi - Company Rules -->
+
+<record model="ir.rule" id="workcenter_group_comp_rule">
+    <field name="name">Workcenter multi-company</field>
+    <field name="model_id" ref="model_workcenter_group"/>
+    <field name="global" eval="True"/>
+    <field name="domain_force">['|',('company_id','=',False),('company_id','child_of',[user.company_id.id])]</field>
+</record>
```
Comment (clonedagain) [inline mrp_workcenter_group/security/model_security.xml:12]: For my own education, isn't this already auto-generated, do we have to do it on all our new models?

File: mrp_workcenter_group/workcenter.py [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+
+    _columns = {
+        'name': fields.char('Name'),
+        'sequence': fields.integer('Sequence'),
+        'active': fields.boolean('Active'),
+        'workcenter_ids': fields.one2many(
+            'mrp.workcenter',
+            'workcenter_group_id',
+            readonly=True,
+            string="Linked Workcenters"
```
Comment (clonedagain) [inline mrp_workcenter_group/workcenter.py]: Maybe just "Workcenters" ?

File: mrp_workcenter_group/security/model_security.xml (hunk 7 above)
Comment (bealdav) [inline mrp_workcenter_group/security/model_security.xml:12]: As far I know not in v7 at least.

**Comment by Viggor** [comment]
Maybe you should add demo data for workcenter groups, because the first time you open workcenters after install this module you have a 'undefined' group, it's a bit confusing.

**Comment by Viggor** [comment]
:+1: for me

**Pull request merged**